packages/hdf5/hdf5.0.1.1/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "Vladimir Brankov <vbrankov@janestreet.com>"
+authors: "Vladimir Brankov <vbrankov@janestreet.com>"
+homepage: "https://github.com/vbrankov/hdf5-ocaml"
+bug-reports: "https://github.com/vbrankov/hdf5-ocaml/issues"
+license: "MIT"
+dev-repo: "git@github.com:vbrankov/hdf5-ocaml.git"
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: [make "install"]
+remove: [
+  ["ocamlfind" "remove" "hdf5_caml"]
+  ["ocamlfind" "remove" "hdf5_raw"]
+]
+depends: [
+  "ocamlfind" {build}
+]
+depexts: [
+  ["ubuntu"]
+  ["libhdf5-serial-dev"]
```

Comment:
It's not strictly necessary but would be less confusing to wrap the Ubuntu depext in `[` and `]` because it *will* be necessary when more are added. The `["osx"] ["homebrew"]` depext is `homebrew/science/hdf5`.